Update new post file and maintainer info

jonatack opened the pull request:
- Add `commit` field and meeting log section to the files generated by the `rake posts:new" task
- Propose steps for adding PR tags and commits in CONTRIBUTING.md
- Propose moving the maintainer info to the bottom of CONTRIBUTING.md

The updated version of CONTRIBUTING.md can be viewed here:
https://github.com/jonatack/bitcoin-core-review-club.github.io/blob/add-tag-commit-hash-info-to-CONTRIBUTING/CONTRIBUTING.md

jnewbery commented on Rakefile:
```diff
@@ -168,9 +168,14 @@ def create_post_file!(filename, response, date, host)
     line.puts "## Notes\n\n"
     line.puts "## Questions\n"
+    line.puts "\n"
+    line.puts "## Meeting Log\n"
```
I think having an empty meeting log before the meeting is ugly. Can you either remove this or comment it out?

jonatack replied:
Good idea. Done.

jonatack commented:
Wrapped the Meeting Log section in HTML comment markup for auto-generated posts, and updated the After-Meeting info in CONTRIBUTING.md to "...uncomment the `## Meeting Log` markup and copy-paste the log..."

jonatack commented:
Re-pushed a couple of fixups to get the CONTRIBUTING docs right.

jnewbery commented:
Thanks Jon. Looks good!

Can you remove the final commit? I think the "Making a New Post" is somewhat separate from the "General Tips", which is why they live under those headings for now. Otherwise, I think this is ready to merge.

jonatack commented:
> Can you remove the final commit? I think the "Making a New Post" is somewhat separate from the "General Tips", which is why they live under those headings for now. Otherwise, I think this is ready to merge.

Agreed, I hesitated on the value of this. Thanks for the feedback! Dropped the last commit and added a typo fix to the second commit. I'll go ahead and merge this.

jnewbery commented:
Thanks Jon! :rocket:
